tortoise/__init__.py

```diff
@@ -495,28 +498,32 @@ async def init(
 
         cls.table_name_generator = table_name_generator
 
-        # Mask passwords in logs output
-        passwords = []
-        for name, info in connections_config.items():
-            if isinstance(info, str):
-                info = expand_db_url(info)
-            if password := info.get("credentials", {}).get("password"):
-                passwords.append(password)
-
-        str_connection_config = str(connections_config)
-        for password in passwords:
-            str_connection_config = str_connection_config.replace(
-                password,
-                # Show one third of the password at beginning (may be better for debugging purposes)
-                f"{password[0:len(password) // 3]}***",
+        if logger.isEnabledFor(logging.DEBUG):
```

Comment:
The new code is overly complex. We shouldn't have something that complicated in the main file of the whole project. I would personally have more stars in the debug logs than 5 nested blocks of code to handle it.